### PR TITLE
Dockerfile: use relative paths for build copy steps

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard24**
Attempt to fix:

Screenshot 2022-04-06 at 11 12 43

### CODE REVIEW

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Quick fix

File: Dockerfile [hunk 1]
```diff
@@ -92,14 +92,14 @@ RUN apk add --no-cache \
 RUN addgroup -g 1001 -S nodejs \
 && adduser -S nextjs -u 1001
 
-COPY --from=builder --chown=nextjs:nodejs /app/packages/app/.next/standalone /app/.next/standalone
-COPY --from=builder --chown=nextjs:nodejs /app/packages/app/.next/static /app/.next/standalone/packages/app/.next/static
-COPY --from=builder /app/packages/app/next.config.js /app/.next/standalone/packages/app
+COPY --from=builder --chown=nextjs:nodejs app/packages/app/.next/standalone app/.next/standalone
+COPY --from=builder --chown=nextjs:nodejs app/packages/app/.next/static app/.next/standalone/packages/app/.next/static
+COPY --from=builder app/packages/app/next.config.js app/.next/standalone/packages/app
```
Comment (VWSCoronaDashboard19) [inline Dockerfile:97]: is `app/packages/app/next.config.js` the correct path? isn't `packages/app/next.config.js` sufficient?

Comment (VWSCoronaDashboard24) [reply Dockerfile:97]: In this case the whole app (with all packages) is copied and built into the /app folder. See https://github.com/minvws/nl-covid19-data-dashboard/pull/4154/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R9 :-)

Comment (VWSCoronaDashboard24) [reply Dockerfile:97]: Or line 9 of the Dockerfile

**Pull request merged**